src/SDSS.jl

```diff
@@ -130,5 +135,454 @@ function load_stamp_catalog(cat_dir, stamp_id, blob; match_blob=false)
 end
 
 
+@doc """
+Load data from a psField file, which contains the point spread function.
+
+Args:
+ - field_dir: The directory of the file
+ - run_num: The run number
+ - camcol_num: The camcol number
+ - frame_num: The frame number
+ - b: The filter band (a number from 1 to 5)
+
+Returns:
+ - rrows: A matrix of flattened eigenimages.
+ - rnrow: The number of rows in an eigenimage.
+ - rncol: The number of columns in an eigenimage.
+ - cmat: The coefficients of the weight polynomial (see below).
+
+The point spread function is represented as an rnrow x rncol image
+showing what a true point source would look as viewed through the optics.
+This image varies across the field, and to parameterize this variation,
+the PSF is represented as a linear combination of four "eigenimages",
+with weights that vary across the image.  See get_psf_at_point()
+for more details.
+""" ->
+function load_psf_data(field_dir, run_num, camcol_num, frame_num, b)
+    @assert 1 <= b <= 5
+    psf_filename = "$field_dir/psField-$run_num-$camcol_num-$frame_num.fit"
+    psf_fits = FITS(psf_filename)
+    psf_hdu = psf_fits[b + 1]
+
+    nrows = read_key(psf_hdu, "NAXIS2")[1]
+    nrow_b = read(psf_hdu, "nrow_b")[1]
+    ncol_b = read(psf_hdu, "ncol_b")[1]
+    rnrow = read(psf_hdu, "rnrow")[1]
+    rncol = read(psf_hdu, "rncol")[1]
+    cmat = convert(Array{Float64, 3}, read(psf_hdu, "c"))
+
+    # Some low-level FITSIO rigamarole until the high-level interface can
+    # read variable length binary rows.
+    
```

Comment:
Yep, I had only pushed a low-level function, but now the whole thing appears to be done.